BoostRunClub/Coordinators/AppCoordinator.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  AppCoordinator.swift
+//  BoostRunClub
+//
+//  Created by Imho Jang on 2020/11/23.
+//
+
+import Combine
+import UIKit
+
+final class AppCoordinator: BasicCoordinator<Void> {
+    override func start() {
+        return showMainFlow()
```

Comment:
```suggestion
        showMainFlow()
```
여기서 return 을 해도 컴파일 에러는 발생하지 않지만 
return 된 Void 값을 사용하지 않기 떄문에 return을 제거하는 것이 가독성에 좋아보입니다.